spring-boot-starter-camunda-sdk/src/main/resources/application-camunda-oidc.yaml

```diff
@@ -5,10 +5,11 @@ camunda:
       - <default>
     auth:
       oidc-type: keycloak
-      issuer: http://localhost:18080/auth/realms/camunda-platform
+      issuer: http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token
```

Comment:
❓ Why this is changed ?

Reply:
This is the URL to obtain JWT token from Keyckloak using standard OAuth protocol.